themes/vocabulary_theme/templates/macros/pagination.html

```diff
@@ -3,23 +3,31 @@
     <ul class="pagination-list">
       {% if pagination.has_prev %}
         <li class="inactive">
-          <a class="pagination-link inactive-link" href="{{ pagination.prev|url }}" aria-label="Goto previous"><i class="icon chevron-left"></i></a>
+          <a class="pagination-link inactive-link" href="{{ pagination.prev|url }}" aria-label="Goto previous page">
+            <i class="icon chevron-left" aria-hidden="true"></i>
+          </a>
         </li>
       {% endif %}
       {% for p in range(pagination.pages) %}
         {% if pagination.current|url == pagination.for_page(p+1)|url %}
           <li class="common">
-            <a class="pagination-link is-current common-link" aria-label="Goto page {{p+1}}" aria-current="page" href="{{ pagination.for_page(p+1)|url}}">{{p+1}}</a>
+            <a class="pagination-link is-current common-link" aria-label="Goto page {{ p+1 }}" aria-current="page" href="{{ pagination.for_page(p+1)|url }}">
```

Comment:
this `anchor` already encloses descriptive text, but the addition of the "go to" might be more descriptive and worth keeping.